src/server/scripts/Northrend/UtgardeKeep/UtgardePinnacle/boss_svala.cpp

```diff
@@ -235,6 +235,7 @@ class boss_svala : public CreatureScript
                 {
                     me->CastSpell(me, SPELL_SVALA_TRANSFORMING1, true);
                     me->UpdateEntry(NPC_SVALA_SORROWGRAVE);
+                    me->SetCorpseDelay(CONFIG_CORPSE_DECAY_ELITE);
```

Comment:
me->SetCorpseDelay(sWorld->getIntConfig(CONFIG_CORPSE_DECAY_ELITE));

Reply:
Is that so?

Reply:
100%, I'll modify just now.

Reply:
but so you had tested it prior to that??